scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -213,6 +220,26 @@ protected static void start(CommandLine commandLine) throws Exception {
         executeStartScripts();
     }
 
+    /**
+     *  Launch boot (i.e., early-starting) microservices (notably, IAM microservice)
+     *
+     */
+    private static void startBootMicroservices() throws IOException, InterruptedException, ExecutionException {
+
+        // Do nothing if PA_home or the boot microservices path is not specified
+        if (!(PASchedulerProperties.SCHEDULER_BOOT_MICROSERVICES_PATH.isSet() &&
+              PASchedulerProperties.SCHEDULER_HOME.isSet()))
+            return;
+
+        // Acquire paths required to start IAM microservice
+        String paHome = PASchedulerProperties.getAbsolutePath(System.getProperty(PASchedulerProperties.SCHEDULER_HOME.getKey()));
+        String microservicesPath = PASchedulerProperties.SCHEDULER_BOOT_MICROSERVICES_PATH.getValueAsString();
```

Comment:
this should be done in one line.

String microservicesPath = PASchedulerProperties.getAbsolutePath( PASchedulerProperties.SCHEDULER_BOOT_MICROSERVICES_PATH.getValueAsString())